tests/conftest.py

```diff
@@ -551,6 +550,8 @@ def deprecated_python() -> bool:
 
 @pytest.fixture(scope="session")
 def cert_factory(tmpdir_factory: pytest.TempdirFactory) -> CertFactory:
+    from tests.lib.certs import make_tls_cert, serialize_cert, serialize_key
```

Comment:
Please add a comment to provide context on why this one import is handled differently from the rest.

Reply:
And a news entry.

Reply:
Done and done.